grails-app/services/org/broadinstitute/orsp/IssueService.groovy

```diff
@@ -237,6 +237,21 @@ class IssueService {
         newIssue
     }
 
+    void deleteIssue(String projectKey) {
+        Issue issue = queryService.findByKey(projectKey)
+        if(issue != null) {
+            def fundingList = queryService.findFundingsByProject(issue.projectKey)
+            fundingList?.each {
+                it.delete()
+            }
+            Collection<IssueExtraProperty> issueExtraProperties = issue.getExtraProperties()
+            issueExtraProperties?.each {
+                it.delete()
+            }
+        }
+        issue.delete(flush: true)
```

Comment:
See https://github.com/broadinstitute/orsp/blob/develop/sql/utility/delete_issue.sql for a full list of things to look at when deleting an issue. And ... thank you for adding this! I run that delete script every other week or so.